gfsphysics/GFS_layer/GFS_driver.F90

```diff
@@ -705,7 +705,7 @@ subroutine GFS_stochastic_driver (Model, Statein, Stateout, Sfcprop, Coupling, &
 ! Since IPD_func0d_proc declares all arguments as intent(inout), we
 ! need to do the same here - however, this way we are loosing the
 ! valuable information on the actual intent to this routine. *DH
-#ifdef __GFORTRAN__
+!#ifdef __GFORTRAN__
```

Comment:
You might as well just remove those blocks entirely ...

Reply:
I can add the code changes in my next commit. Let's commit it as is.

Reply:
@climbfuji Do you want me to leave your comments about arguments mismatch or delete them?

Reply:
I think it is best to delete the comments when we remove the code that you commented out entirely.